src/test/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/ValuationInformationPolicyServiceImplTests.java

```diff
@@ -0,0 +1,153 @@
+import uk.gov.companieshouse.web.accounts.model.smallfull.notes.accountingpolicies.ValuationInformationPolicy;
+import uk.gov.companieshouse.web.accounts.service.smallfull.AccountingPoliciesService;
+import uk.gov.companieshouse.web.accounts.service.smallfull.ValuationInformationPolicyService;
+import uk.gov.companieshouse.web.accounts.transformer.smallfull.AccountingPoliciesTransformer;
+import uk.gov.companieshouse.web.accounts.validation.ValidationError;
+
+@ExtendWith(MockitoExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class ValuationInformationPolicyServiceImplTests {
+
+    private static final String TRANSACTION_ID = "transactionId";
+
+    private static final String COMPANY_ACCOUNTS_ID = "companyAccountsId";
+
+    private static final String VALUATION_INFORMATION_POLICY_FIELD_PATH =
+            "valuationInformationPolicyDetails";
+
+    private static final String INVALID_STRING_SIZE_ERROR_MESSAGE =
+            "validation.length.minInvalid.accounting_policies.valuation_information_and_policy";
+
+    @Mock
+    private List<ValidationError> apiErrors;
+
+    @Mock
+    private AccountingPoliciesService accountingPoliciesService;
+
+    @Mock
+    private AccountingPoliciesTransformer accountingPoliciesTransformer;
+
+    @InjectMocks
+    private ValuationInformationPolicyService service = new ValuationInformationPolicyServiceImpl();
+
+    @Test
+    @DisplayName("Get valuation information policy")
+    void getValuationInformationPolicy() throws ServiceException {
+
+        AccountingPoliciesApi accountingPoliciesApi = new AccountingPoliciesApi();
+        when(accountingPoliciesService.getAccountingPoliciesApi(TRANSACTION_ID, COMPANY_ACCOUNTS_ID))
+                .thenReturn(accountingPoliciesApi);
+        when(accountingPoliciesTransformer.getValuationInformationPolicy(accountingPoliciesApi))
+                .thenReturn(new ValuationInformationPolicy());
+
+        ValuationInformationPolicy valuationInformationPolicy =
+                service.getValuationInformationPolicy(TRANSACTION_ID, COMPANY_ACCOUNTS_ID);
+
+        assertNotNull(valuationInformationPolicy);
+    }
+
+    @Test
+    @DisplayName("Submit invalid valuation information policy")
+    void submitInvalidValuationInformationPolicy() throws ServiceException {
+
+        ValuationInformationPolicy valuationInformationPolicy = createInvalidValuationInformationPolicy();
+
+        List<ValidationError> validationErrors =
+                service.submitValuationInformationPolicy(TRANSACTION_ID, COMPANY_ACCOUNTS_ID, valuationInformationPolicy);
+
+        assertEquals(1, validationErrors.size());
+        assertEquals(INVALID_STRING_SIZE_ERROR_MESSAGE, validationErrors.get(0).getMessageKey());
+        assertEquals(VALUATION_INFORMATION_POLICY_FIELD_PATH, validationErrors.get(0).getFieldPath());
+    }
+
+    @Test
+    @DisplayName("Submit  valuation information policy not provided")
```

Comment:
There's a stray space character in the test description here between the words `Submit` and `valuation`.